notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -1611,6 +1611,56 @@ reconciliation [Mark was inspired by Prot to reconcile with his own estranged so
 cyclical time universe [According to Prot, The universe was destined to expand, collapse back on itself, and then expand again in a forever repeating process.]
 
 
+movie: Impostor (2001)
+======================
+
+:: Title
+Impostor
+
+:: Description
+Impostor is a 2002 American science fiction psychological thriller film based
+upon the 1953 short story "Impostor" by Philip K. Dick.
+
+Synopsis: A famous weapons designer is accused of being an alien designed
+replicant and goes on the run from the authorities.
```

Comment:
I guess "replicant" ought to be quoted as it has a meaning specific to this story. The sentence took a re-read to parse.

Reply:
Sure, no objection. But better not to merge the pull request until after such changes are effected.